### ISSUE TITLE
Don't see the 'refreshing' popup when chat comments can't connect

### ISSUE THREAD

**Issue by karlorg** [karlorg/drunken-octo-avenger#63]
I tried loading a game page, stopping the local server and posting a comment.  No visible effect, but from the code it looks like there should be a popup of some kind?

**Comment by allanderek** [comment]
No, there should be no pop-up. The place where it says "Refreshing comments" should change to "Error: could not contact server."

**Comment by karlorg** [comment]
On 26/03/2016 11:03, allanderek wrote:
> No, there should be no pop-up. The place where it says "Refreshing
> comments" should change to "Error: could not contact server."

Where should this appear?  I looked around and couldn't see either message.

**Comment by allanderek** [comment]
I'm definitely seeing the 'Refreshing Comments' appear. It appears top-right of the chat-box. Basically to the right of where the 'Refresh Comments' button is.
